packages/service/src/pool/live-pool-service.ts

```diff
@@ -20,10 +19,7 @@ const SINGLE_POOL_PATH =
 const POOLS_PATH =
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}/pools";
 
```

Comment:
Good call - I'm not sure there's any reason to keep AbstractHttpService at this point.